vms/job/tests/test_searchJob.py

```diff
@@ -0,0 +1,457 @@
+                                search_page.get_search_results)
+
+        search_page.search_state_field('!@#$%^&*()_')
+        search_page.submit_form()
+        self.assertNotEqual(search_page.get_help_block(), None)
+
+    def test_job_country_field(self):
+        """
+        Test search results for job country
+        """
+        search_page = self.search_page
+        search_page.live_server_url = self.live_server_url
+
+        credentials_1 = ['event', '2050-05-10', '2050-06-16']
+        event_1 = create_event_with_details(credentials_1)
+        event_1.country = 'job-country'
+        event_1.save()
+        credentials_1 = ['job-name', '2050-06-10', '2050-06-11', 'job-description', event_1]
+        job_1 = create_job_with_details(credentials_1)
+
+        credentials_2 = ['eventq', '2050-05-10', '2050-06-16']
+        event_2 = create_event_with_details(credentials_2)
+        event_2.country = 'job-countryq'
+        event_2.save()
+        credentials_2 = ['job-nameq', '2050-05-15', '2050-05-20', 'job-description', event_2]
+        job_2 = create_job_with_details(credentials_2)
+
+        expected_result_one = ['job-name', 'event', 'June', '10,', '2050', 'June', '11,', '2050', 'Details', 'Edit', 'Delete']
+        expected_result_two = ['job-nameq', 'eventq', 'May', '15,', '2050', 'May', '20,', '2050', 'Details', 'Edit', 'Delete']
+
+        search_page.navigate_to_job_search_page()
+        search_page.search_country_field('job')
+        search_page.submit_form()
+        search_results = search_page.get_search_results()
+        result = search_page.get_results_list(search_results)
+        self.assertEqual(len(result), 2)
+        self.assertTrue(expected_result_one in result)
+        self.assertTrue(expected_result_two in result)
+
+        search_page.search_country_field('j')
+        search_page.submit_form()
+        search_results = search_page.get_search_results()
+        result = search_page.get_results_list(search_results)
+        self.assertEqual(len(result), 2)
+        self.assertTrue(expected_result_one in result)
+        self.assertTrue(expected_result_two in result)
+
+        search_page.search_country_field('job-')
+        search_page.submit_form()
+        search_results = search_page.get_search_results()
+        result = search_page.get_results_list(search_results)
+        self.assertEqual(len(result), 2)
+        self.assertTrue(expected_result_one in result)
+        self.assertTrue(expected_result_two in result)
+
+        search_page.search_country_field('job-fail-test')
+        search_page.submit_form()
+        self.assertRaisesRegexp(NoSuchElementException,
+                                'Unable to locate element: //table//tbody',
+                                search_page.get_search_results)
+
+        search_page.search_country_field('!@#$%^&*()_')
+        search_page.submit_form()
+        self.assertNotEqual(search_page.get_help_block(), None)
+
+
```

Comment:
In a class, there should be one blank line between functions. Check for this inconsistency.